<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>报文信息</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>已建立连接</string>
       </property>
       <widget class="QTableWidget" name="twConnected">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>本机IP：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pbClear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leIPAddr"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lePort"/>
     </item>
     <item row="3" column="2">
      <widget class="QListWidget" name="lwLog"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbInitServer">
       <property name="text">
        <string>初始化服务端</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
